### PR TITLE
refactor(compiler): Remove strictStyling optionfor ShadowCss

### PR DESCRIPTION
**Pull request by JeanMeche**
`strictStyling` was an option provided by Polymer but it's not needed by Angular. This commit removes the dead code and update the comments.

## PR Type
What kind of change does this PR introduce?

- [x] Refactoring (no functional changes, no api changes)

## Does this PR introduce a breaking change?

- [x] No

### CODE REVIEW

**Comment by JeanMeche** [comment]
Poke @dario-piotrowicz you might be interested by this 👍

**Review by dylhunn — APPROVED** [review]

**Comment by dylhunn** [comment]
Oh wait, I'm actually not in the `fw-compiler` review group. Perhaps I should be though, but for now you'll need a compiler approver

**Comment by dario-piotrowicz** [comment]
> Poke @dario-piotrowicz you might be interested by this +1

Yeah, thanks! :smiley:

**Review by dario-piotrowicz — APPROVED** [review]
@JeanMeche awesome I really like it! :heart_eyes: 

a small nitpick, the commit has a couple of typos :slightly_smiling_face: 
```diff
-refactor(compiler): Remove strictStyling optionfor ShadowCss
+refactor(compiler): Remove strictStyling option for ShadowCss
- `strictStyling` was an option provided by Polymer but it's not needed by Angular. This commit removes the dead code and update the comments.
+ `strictStyling` was an option provided by Polymer but it's not needed by Angular. This commit removes the dead code and updates related comments.
```

File: packages/compiler/src/shadow_css.ts [hunk 1]
```diff
@@ -29,8 +29,6 @@ const animationKeywords = new Set([
 /**
  * The following class is a port of shadowCSS from webcomponents.js to TypeScript.
  *
- * Please make sure to keep to edits in sync with the source file.
- *
  * Source:
  * https://github.com/webcomponents/webcomponentsjs/blob/4efecd7e0e/src/ShadowCSS/ShadowCSS.js
  *
```
[the remark this answers is not in the document]
Comment (JeanMeche) [reply packages/compiler/src/shadow_css.ts]: good idea, I changed it a bit !

**Review by atscott — APPROVED** [review]

**Comment by atscott** [comment]
caretaker note: Windows CI filesystem failures are known flakes

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit c2bcf0b47540510a24931da7cd24f58b60788881.

**Pull request closed without merging**